include/pgduckdb/pgduckdb_duckdb.hpp

```diff
@@ -10,7 +10,30 @@ extern "C" {
 
 namespace pgduckdb {
 
-duckdb::unique_ptr<duckdb::DuckDB> DuckdbOpenDatabase();
+class DuckDBManager {
+public:
+	static inline const DuckDBManager &
+	Get() {
+		static DuckDBManager instance;
+		return instance;
+	}
+
+	inline duckdb::DuckDB &
+	GetDatabase() const {
+		return *database;
```

Comment:
Can we do a nullcheck here? I imagine this just causes a generic looking InternalException if `database` is null which will be hard to debug